src/main/java/uk/gov/companieshouse/web/accounts/validation/smallfull/EmployeesValidator.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.companieshouse.web.accounts.validation.smallfull;
+
+import org.apache.commons.lang.StringUtils;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.employees.AverageNumberOfEmployees;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.employees.Employees;
+
+public class EmployeesValidator implements Validator {
+
+    @Override
+    public boolean supports(Class clazz) {
+      return Employees.class.equals(clazz);
+    }
+
+    @Override
+    public void validate(Object target, Errors errors) {
+        Employees employees = (Employees) target;
+
+      if (!isEmployeesValid(employees) && !errors.hasFieldErrors()) {
+
+          errors.reject("validation.missing.employees", new Object[]{}, null);
+      }
+    }
+
+    private boolean isEmployeesValid(Employees employees) {
+
+        if (employees == null || (isAverageNumberOfEmployeesNull(employees) &&
+               StringUtils.isEmpty(employees.getDetails()))) {
+            return false;
+        } else {
+            return true;
+        }
```

Comment:
This conditional here can be compressed to a single `return` statement and will silence any associated sonar warnings:

```java
return employees == null || (isAverageNumberOfEmployeesNull(employees) &&
               StringUtils.isEmpty(employees.getDetails());
```